jdk/make/CopyFiles.gmk

```diff
@@ -23,6 +23,23 @@
 # questions.
 #
 
+# ===========================================================================
```

Comment:
Please put the IBM copyright above the license above

Reply:
Updated.